MaraBot/Program.cs

```diff
@@ -19,7 +21,12 @@ static async Task MainAsync()
         {
             var config = ConfigIO.LoadConfig();
             var presets = PresetIO.LoadPresets();
-            
+			using (StreamReader r = new StreamReader("config/options.json"))
+			{
+				var json = r.ReadToEnd();
+				Option.Options = JsonConvert.DeserializeObject<Dictionary<string, Option>>(json);
+			}
+
```

Comment:
Let's move this to a dedicated `OptionsIO.cs` so that we keep everything in `Program.cs` nice and tidy.